Makefile

```diff
@@ -1,35 +1,37 @@
-.DEFAULT_GOAL := interactive
-.DEV_IMAGE := data-explorer-sa
-.SERVE_IMAGE := data-explorer-serve
+.DEFAULT_GOAL		:= interactive
 
-.PORT := 8081
-.PORT_DEBUG := 9229
-.DB_HOSTNAME := host.docker.internal

```

Comment:
```
ifeq ($(shell uname), Darwin)
    # macOS
    .DB_HOSTNAME := host.docker.internal
else ifeq ($(shell uname), Linux)
    # Linux
    .DB_HOSTNAME := $(shell hostname -I | cut -d ' ' -f 1)
else
    # Default for other systems (assuming Windows)
    .DB_HOSTNAME := host.docker.internal
endif
```
sadly linux dont support this alias, may be this work arround will help

Reply:
applied